specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,539 @@
+in ux, when considering batch size request and gas limitations, that might trip some developers up. 
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+The rationale for this design, is that we want to support developers that need batch requests. In many ecosystems, a single NFT or 
+series of NFT are not sufficient for representing and managing potentially 100s of tokens. Managing cross contract calls and the token contracts themselves become a huge burden on the developer. This will reduce the complexity, and allow developers to easily bridge over to other contracts like ERC-1155 on other chains that allow for a this style of representation.
+
+In the design phase it was considered to simply rely on the underlying implementations of fungible tokens and non-fungible tokens
+to be the scope of interaction with the chain. Doing this we would have tied the implementations to the bounds of FT and NFT. By loosing this up a bit, we are able to be a bit more flexible in what's possible.
+
+Not doing this means we really won't have a great way of supporting use cases where developers need to represent and manage
+large varying quantities and types of tokens. They will all have to implement it on their own, and that would make for a fragmented
+and weak ecosystem. Where every developer would not be able to reliably trade these assets.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+The unresolved questions, are really what type of metadata , is required for this spec?
+- We decided that having metadata on chain is good, and that at later time we can have a spec for extra data if needed
+
+Can we represent events in this spec, which would improve the ecosystem quite a bit? If we represent events what should those events be?
+- We decided events aren't here yet in the ecosystem, in the way we' like them to be so they are't apart of this standard
+
+Should we have a spec for TokenType?
+- We decided TokenType shouldn't be exposed for the public interface consumption. We will leave this description in the hands of the implementers
+
+Should we have a spec for offchain metadata?
+- We decided no spec for offchain metadata yet but maybe in the future
+
+Does the current storage management scheme work for people?
+- The current storage management scheme works for folks
+
+How freeform should this token be? Right now there is a notion of supply, which is not 100% guaranteed every token has
+or wants to track supply semantics. Not having supply makes everything more difficult and requires consumers of the contract
+to track minting and burning events.
+- This point was taken to show that due to lack of proper events, and having the additional capability of storing this on chain, would
+result in a reduction of complexity for consumers of the contract data.
```

Comment:
Could you clarify what we mean with these statements?  The last bullet point in particular here sounds like a jumble of concepts and I don't understand what it's trying to say; what is `this`. in this sentence?  It feels like this should be a more direct statement like "Should this standard support the concept of a 'supply-less' token?", with a definition of what 'supply-less' means and the reasons why we have avoided adding it.  It's not clear how having events or an 'additional capability of storing this on chain' (i'm not sure what this means) would make a difference to this decision (or if it would at all)

Reply:
These statements are questions, that we didn't have answers to in the beginning. 
The result was to be a quick 1-2 line explanation of why we chose what we chose. There's alot of depth in the additional meeting notes. 

It's possible this is just not communicating what I'd like and it's just better to not have this section. Alternatively, beef it out to be more verbose, though a lot of the discussion I believe is captured in meeting notes.

Reply:
The meeting notes having more detail is great, but I think the idea behind this as a published standard is to have the relevant information about that standard included in the standard's documentation itself.  I don't think that all of the notes from every meeting should be here, but I think that the standard should clearly describe the conclusions made as a result of those discussions, and the reasoning behind those conclusions -- Imagine you are reading this specification for the first time as a developer who is trying to decide whether to use it or another NEP, and how to use it for their use cases.

Reply:
These have been resolved in the rewrite of documentation to match the other standards